packages/shared/src/components/streak/ReadingStreakButton.tsx

```diff
@@ -28,6 +28,7 @@ function CustomStreaksTooltip({
       interactive
       showArrow={false}
       visible={shouldShowStreaks}
+      forceLoad
```

Comment:
Tests will fail because of Tippy.js if this was included in any unit file. You need `forceLoad={!isTesting}`.

Reply:
thx for the pointer 🙏